data/src/main/java/org/oppia/android/data/backends/gae/model/GaeFeedbackReport.kt

```diff
@@ -0,0 +1,20 @@
+package org.oppia.android.data.backends.gae.model
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+/**
+ * Data class for the feedback report sent by the Android app to remote storage.
+ * TODO(#2801): Link backend domain model
+ */
+@JsonClass(generateAdapter = true)
+data class GaeFeedbackReport constructor(
```

Comment:
Strike 'constructor'--it's not needed since there's nothing being modified about it (e.g. visibility or annotations).

Reply:
Done